modular_skyrat/modules/automapper/code/automapper_subsystem.dm

```diff
@@ -17,7 +17,7 @@ SUBSYSTEM_DEF(automapper)
 	flags = SS_NO_FIRE
 	init_order = INIT_ORDER_AUTOMAPPER
 	/// The path to our TOML file
-	var/config_file = "_maps/skyrat/automapper/automapper_config.toml"
+	var/config_file = "_maps/bubber/automapper/automapper_config.toml" //BUBBER EDIT: POINTS TO OUR MODULAR AUTOMAPPER
```

Comment:
What if Skyrat adds a new automapper template or otherwise changes their config? I feel like we should just edit the config non-modularly like we do with TGUI, but input on this would be appreciated!

Reply:
So- if we need to edit existing skyrat ones we can do non-modular changes, but for pure additions... we could add another automapper config toml for us, and then at compiletime concatenate the two together(this is trivial) into an effective config.

Reply:
That would be nice, yes